model/note/note_test.go

```diff
@@ -46,7 +58,15 @@ hello world
 
 - bar
 
-- baz`
+- baz
+
+This is a decision
+
+and another decision
+
+- A todo task
+
+- a done task`
```

Comment:
Where do we need this `Text` stuff where we have a markdown as input?

Reply:
We don't. We want to test the `Text` output of a note, and it is too painful to create a note with its prosemirror tree manually to do that, so we import a markdown as a way to bootstrap a prosemirror tree in this test.

Reply:
And where do we need this `Text` output of a note? I'm asking in order to understand to not forget to update the test if I try to add a few plugins later.

Reply:
For the `MesPapiers` app, to copy the text of the notes and allow to paste it somewhere else

Reply:
I don't understand why we strip the `✍` character and `[ ]` or `[x]` strings in the text version.
These are perfectly understandable in their text form and removing them, especially the checkboxes, removes important information.

Reply:
The requirement is to remove all markup. The only exception is `-` for lists.

Reply:
Ah OK.